statsmodels/stats/multitest.py

```diff
@@ -10,6 +10,7 @@
 import numpy as np
 
 from statsmodels.stats._knockoff import RegressionFDR
+from sklearn.isotonic import isotonic_regression
```

Comment:
sklearn dependency is a problem because we don't require it
recent scipy also has a version, which AFAIK was contributed by sklearn developers
https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.isotonic_regression.html

otherwise we could add a copy of the relevant code in statsmodels